pom.xml

```diff
@@ -47,6 +47,12 @@
     </dependency>
 
     <dependency>
+      <groupId>org.slf4j</groupId>
```

Comment:
Do we need to add this as a hard dependency? Does maven have a way to "suggest" dependencies?